src/applications/hca/components/VeteranInformation/GuestVerifiedInformation.jsx

```diff
@@ -25,28 +25,33 @@ const GuestVerifiedInformation = ({ user }) => {
   return (
     <div className="vads-u-margin-top--2p5 vads-u-margin-bottom--2">
       <p>Confirm your information before you continue.</p>
-      <va-card data-testid="hca-guest-card" background>
+      <va-card data-testid="hca-guest-card">
+        <h4 className="vads-u-font-size--h3 vads-u-margin-top--0">
```

Comment:
This should be either an H2 or H3 depending on whether the "Confirm your information..." statement above turns to a heading or not.